doc/UserGuide.md

```diff
@@ -41,6 +41,8 @@ Format: `list`
 ## Finding all persons containing any keyword in their name: `find`
```

Comment:
The title should be changed since the functionality of _find_ has been expanded.